### PR TITLE
Change sbt welcome logo to Pekko

### PR DESCRIPTION
**Pull request by esamson**

### CODE REVIEW

**Review by Claudenw — APPROVED** [review]
I see this change as a simple code change more akin to a documentation change.

I don't see anything that needs changed.

**Review by pjfanning — APPROVED** [review]
Seems fine to me - I haven't tried the change but it should be easy to fix if it doesn't appear as expected

**Review by mdedetrich — APPROVED** [review]
Nice work!

File: project/AkkaBuild.scala [hunk 1]
```diff
@@ -266,12 +266,12 @@ object AkkaBuild {
...
-           |___    |___  /_____  /________ _
-           |__  /| |__  //_/__  //_/_  __ `/
-           |_  ___ |_  ,<   _  ,<   / /_/ /
-           |/_/  |_|/_/|_|  /_/|_|  \\__,_/   ${version.value}
+        raw"""
+           |________     ______ ______        
+           |___  __ \_______  /____  /_______ 
+           |__  /_/ /  _ \_  //_/_  //_/  __ \
+           |_  ____//  __/  ,<  _  ,<  / /_/ /
+           |/_/     \___//_/|_| /_/|_| \____/   ${version.value}
```
Comment (He-Pin) [inline project/AkkaBuild.scala:274]: Cool~~

**Review by nvollmar — APPROVED** [review]

**Review by andreaTP — APPROVED** [review]

**Review by seglo — APPROVED** [review]
Nice. I was actually looking at some figlet alternatives, graffiti is my favourite ;)

http://www.network-science.de/ascii/

**Comment by pjfanning** [comment]
seems safe to merge now - thanks

**Pull request merged**